cypress/support/viewUtils.ts

```diff
@@ -108,7 +108,8 @@ export const expectLinkViewScreenLayout = ({
   }
 
   if (!html && (settings?.showLinkButton ?? DEFAULT_LINK_SHOW_BUTTON)) {
-    cy.get('[data-testid="OpenInNewIcon"]').should('be.visible');
+    // this data-testid is set in graasp/ui
+    cy.get('[data-testid="fancy-link-card"]').should('be.visible');
```

Comment:
Is the UI exposed a constant for that ?

Reply:
UI has a constant for that, but I am not sure we should use it in the test because:
- It is possible that cypress will try to bundle all of ui for the tests even if we only use a constant
- I am not sure we want to depend on the constant, I think there should be a better way to test the presence of the card ... 

@ReidyT Is it ok if we keep it like this ?

Reply:
Thanks for your response. Yes, I think you can let it like that, especially if we risk to bundle all UI just for this... It was more a question 😄. Normally this constant in UI shouldn't changed frequently so it is ok.